app/models/package_manager/go.rb

```diff
@@ -50,9 +50,9 @@ def self.mapping(project)
     end
 
     # https://golang.org/cmd/go/#hdr-Import_path_syntax
-    def self.resolved_name(name)
-      return name if name.start_with?(*KNOWN_HOSTS)
-      return name if KNOWN_VCS.any?(&name.method(:include?))
+    def self.project_find_names(name)
+      return [name] if name.start_with?(*KNOWN_HOSTS)
```

Comment:
You could condense this down to one line although it would be quite dense 😄

Reply:
This could all be much cleaner if `PackageManager` classes were instantiated instead of all class methods :/